cleaners/teamviewer.xml

```diff
@@ -33,8 +33,8 @@
 <cleaner id="teamviewer" os="windows">
   <label>TeamViewer</label>
   <description>Remote control software</description>
-  <var name="profile">
-    <value os="windows">$AppData\TeamViewer</value>
+  <var name="Profile">
+    <value search="glob" os="windows">%AppData%\TeamViewer</value>
```

Comment:
`search="glob"` is unnecessary here because `%AppData%\TeamViewer` contains no wildcard

Reply:
I remove now all `search="glob"` from `<value>` where it isn't necessary in my cleaners...